src/shape/shape.rs

```diff
@@ -24,17 +24,30 @@ impl<N: RealField, T: 'static + Shape<N> + Clone> ShapeClone<N> for T {
 ///
 /// This allows dynamic inspection of the shape capabilities.
 pub trait Shape<N: RealField>: Send + Sync + Downcast + ShapeClone<N> {
-    /// The AABB of `self`.
+    /// The AABB of `self` transformed by `m`.
     #[inline]
     fn aabb(&self, m: &Isometry<N>) -> AABB<N>;
 
-    /// The bounding sphere of `self`.
+    /// The AABB of `self`.
+    #[inline]
+    fn local_aabb(&self) -> AABB<N> {
+        self.aabb(&Isometry::identity())
+    }
+
+    /// The bounding sphere of `self` transformed by `m`.
     #[inline]
     fn bounding_sphere(&self, m: &Isometry<N>) -> BoundingSphere<N> {
         let aabb = self.aabb(m);
         BoundingSphere::new(aabb.center(), aabb.half_extents().norm_squared())
```

Comment:
It turns out there was a bug in the existing implementation. This should fix it.
```suggestion
        BoundingSphere::new(aabb.center(), aabb.half_extents().norm())
```